### DISCUSSION TITLE
Add support for Aeratron Fans

### DISCUSSION THREAD

**Discussion by sschnoor** [tuya/tuya-homebridge#373] [General]
Fan on/off is currently supported, but when attempting to change the fan speed an error is thrown : 

_This plugin generated a warning from the characteristic 'Brightness': characteristic was supplied illegal value: number -2 exceeded minimum of 0._

This error happens anytime you attempt to adjust the fan speed up or down.

Please add support for Aeratron!  Thank you.

**Comment by ChrisJS69** [comment]
Thinking about buying an Aeratron fan but not being able to operate it via Home is a deal breaker. Was your fan speed issue ever resolved?
